core/cli/src/lib.rs

```diff
@@ -416,17 +373,17 @@ where
 	if let Some(matches) = matches.subcommand_matches("build-spec") {
 		build_spec::<F>(matches, spec, config)?;
 		return Ok(Action::ExecutedInternally);
-	} else if let Some(sub_matches) = matches.subcommand_matches("export-blocks") {
+	} else if let Some(_sub_matches) = matches.subcommand_matches("export-blocks") {
 		export_blocks::<F, _>(
-			get_db_path_for_subcommand(matches, sub_matches, app_info)?,
+			&config.database_path,
 			matches,
```

Comment:
Could you please replace `matches` with `sub_matches`. We should clearly forward here the matches of the sub-command. And could you please do that for the other sub-commands as well.

Reply:
Fixed.